.github/workflows/verify.yml

```diff
@@ -0,0 +1,57 @@
+name: verify
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - release-*
+  pull_request:
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.15'
+
+      - uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.32.2
+
+          # Only show new issues for a pull request.
+          only-new-issues: true
```

Comment:
Let's cherry-pick this commit to fix the CI: https://github.com/cri-o/cri-o/pull/4971/commits/332ce77615ae0d9c124e211b36fce976efa10050